dist2src/core.py

```diff
@@ -159,7 +160,13 @@ def cherry_pick_base(self, from_branch, to_branch, theirs=False):
             if theirs
             else {}
         )
-        self.repo.git.cherry_pick(f"{from_branch}~{num_commits - 1}", **git_options)
+        try:
+            self.repo.git.cherry_pick(f"{from_branch}~{num_commits - 1}", **git_options)
+        except GitCommandError as ex:
+            if "nothing to commit" in str(ex):
+                self.commit(message="Base commit: empty - no source archive")
```

Comment:
Do we want to have it explicit?

We could use `--allow-empty` as well.

Reply:
we use --allow-empty, see definition for commit above

Reply:
I mean the `--allow-empty` in the `cherry_pick` command.

Reply:
ahaaaaaa

Reply:
sadly it still doesn't work:
```
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git cherry-pick --allow-empty updates~0
  stderr: 'error: could not resolve HEAD commit
fatal: cherry-pick failed'
```